src/Note.ts

```diff
@@ -54,7 +54,13 @@ class Note implements PlayableInterface {
     let oscillatorNode: OscillatorNode;
     if(oscillator === 'piano') {
       oscillatorNode = this.getPianoOscillator(audioContext);
-    } else {
+    } else if(oscillator === 'betterpiano') {
+      oscillatorNode = this.getBetterPianoOscillator(audioContext);
+    } else if(oscillator === 'bestpiano') {
+      oscillatorNode = this.getBestPianoOscillator(audioContext);
+    } else if(oscillator === 'guitar') {
+      oscillatorNode = this.getGuitarOscillator(audioContext);
+    }else {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏garde que piano et guitar